### PR TITLE
[Staging] fix : add hover in different places [123]

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

Hover effect on tab bar

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte [hunk 1]
```diff
@@ -251,5 +256,21 @@
...
+    border-radius: 2px;
+  }
+  .plus-btn:hover{
+    background-color: #1E1E1E;
+    border-radius: 2px;
+  }
+  .layout:hover{
+    background-color: #1E1E1E;
+    border-radius: 2px;
+  }
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte:274]: use color from variable

File: src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte [hunk 2]
```diff
@@ -125,21 +127,21 @@
...
+    />
+        <button on:click={() => {
+          tabBarScroller("right");
+        }} role="button" 
+        class=" btn border-0  ps-1 pe-1 py-auto h-100 w-100"  
+        style=" width:20px; transform: rotate(180deg); margin: 0 !important; height:22px;">
+          <div class="right-btn d-flex pt-1 pb-1 justify-content-center align-items-center"  
+          style="height: 22px; width:22px;">
+            <AngleLeft height={"12px"} width={"24px"} />  
+          </div>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/tab-bar/layout/TabBar.svelte:143]: pass color props

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**